tests/unit/libstore/legacy-ssh-store.cc

```diff
@@ -0,0 +1,22 @@
+#include <gtest/gtest.h>
+
+#include "legacy-ssh-store.hh"
+
+namespace nix {
+
+TEST(LegacySSHStore, constructConfig)
+{
+    LegacySSHStoreConfig{
```

Comment:
do you want to test for equality or something rather than just creation?
or get out the values individually?